packages/styled/base/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "main": "dist/styled.cjs.js",
+  "module": "dist/styled.esm.js",
+  "umd:main": "dist/styled.umd.min.js",
+  "browser": {
+    "./dist/styled.cjs.js": "./dist/styled.browser.cjs.js",
```

Comment:
i'm not entirely sure about it right now - but I vaguely remember at least some tools ignoring this alias field for extra entries, and that I had to include those redirects in root package.json